IEEE Xplore.js

```diff
@@ -518,8 +530,7 @@ var testCases = [
 				],
 				"date": "January 1991",
 				"DOI": "10.1109/78.80767",
-				"ISSN": "1053-587X",
```

Comment:
Why is the ISSN now missing? Did you also test the other examples or only this one?

Reply:
To be honest, I only test my code by visiting `https://ieeexplore.ieee.org/xpl/conhome/7048058/proceeding` and try to downloads all papers. The page was detected by the translator and I downloads all papers from the conference successfully. So I think my code work correctly. The only thing my code will do is recognizing pages whose url matches `xpl/conhome/\\d+/proceeding` as `multiple`. It won't disturb other test cases. 

I tried to pass all test cases. But when I click `run` button of Scaffold, it always report "TranslatorTester: IEEE Xplore Test 1: failed (Detection failed)" (for all of tests). I have no idea why this happens. When I paste the url of a test(`https://ieeexplore.ieee.org/document/80767/`) to `Browser` tab, everything works fine. I click `Test Regex`, it  outputs `===>true<=== (boolean)` . Then I click `Run detectWeb`, it outputs `detectWeb returned type "journalArticle"`. It seems that the detection process is correct. Why it output `Detection failed` when I click `run` ? I have no idea, so I give up.

I think the reason of why the ISSN is missing is that the procedure of parsing IEEE web pages is outdated. Maybe the position of some elements of the web page is changed. I'll try my best to handle this if needed.

Reply:
> Why is the ISSN now missing? Did you also test the other examples or only this one?

After some debug, I found that ISSN is missing because IEEE Xplorer no longer provide it. When I tried to click `Download citation` and select `BibTex` output format manually, I got:
```
@ARTICLE{80767,
author={A. M. {Haimovich} and Y. {Bar-Ness}},
journal={IEEE Transactions on Signal Processing},
title={An eigenanalysis interference canceler},
year={1991},
volume={39},
number={1},
pages={76-84},
keywords={adaptive filters;eigenvalues and eigenfunctions;filtering and prediction theory;interference suppression;signal processing;eigenanalysis methods;adaptive implementation;interference cancellation;array processing;fast converging algorithm;Interference cancellation;Background noise;Sensor arrays;Signal to noise ratio;Jamming;Array signal processing;Steady-state;Noise cancellation;Direction of arrival estimation;Optimization methods},
doi={10.1109/78.80767},
ISSN={},
month={Jan},}
```
As can be seen above, `ISSN` is missing. `IEEE Xplorer.js` extract `ISSN` information from the downloaded BibTex citation. Thus, the ISSN is now missing because it is missing in the downloaded BibTex citation.

Reply:
Okay, I looked into this and tried to extract the ISSN from other data in the website. The tests for multiple within Scaffold might fail due to some dynamic content loading.

Reply:
Thanks ^_^